src/foundation/heading.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Heading = ({
+  children,
+  className,
+  level,
+  display,
+  tag: Tag = `h${level}`,
+  ...restProps
+}) => (
+  <Tag className={`${display ? 'display' : ''} ${className}`} {...restProps}>
```

Comment:
Would be nice to start using `classNames`

Reply:
I would add it later